app/components/plugin.tsx

```
@@ -334,26 +340,28 @@ export function PluginPage() {
+                      'markdown-body',
+                      pluginStyles['plugin-content'],
                     )}
                     dir="auto"
                   >
                     <pre>
                       <code
-                        contentEditable={true}
+                        contentEditable
                         dangerouslySetInnerHTML={{
                           __html: editingPlugin.content,
                         }}
```

Biome (1.9.4)

[error] Avoid passing both children and the dangerouslySetInnerHTML prop.

This is the source of the children prop

Setting HTML content will inadvertently override any passed children in React

(lint/security/noDangerouslySetInnerHtmlWithChildren)